### PR TITLE
Add a note about macOS Big Sur incompatibility

### PR DESCRIPTION
**Pull request by emilmelnikov**

### CODE REVIEW

File: download.html [hunk 1]
```diff
@@ -13,6 +13,16 @@ <h2>Binaries</h2>
 <br>
 </p>
 
+<div class="panel panel-danger">
+  <div class="panel-heading">
+    <h3 class="panel-title">macOS Big Sur is not supported</h3>
+  </div>
+  <div class="panel-body">
+    If you are using macOS, please <em>do not upgrade</em> if ilastik is critical to your workflow.
```
Comment (k-dominik) [inline download.html:21]: upgrade what? ilastik or OSX?

```suggestion
    If you are using macOS, please <em>do not upgrade to macOS 11.0 (Big Sur)</em> if ilastik is critical to your workflow.
```

**Review by k-dominik — COMMENTED** [review]
cool I like the appearance of this little danger panel :), but I think wording could be clarified as we are on the "ilastik upgrade" page, so upgrade would usually refer to as new ilastik versions.

**Comment by k-dominik** [comment]
@akreshuk, we thought during the meeting that it might be a good idea to warn our OSX users via twitter about this?

File: download.html (hunk 1 above)
Comment (emilmelnikov) [reply download.html:21]: I'll drop `.0` from `11.0` (what if they'll release `11.1` tomorrow?..)

**Pull request merged**